src/modules/Favorites/components/FavoriteAlbums.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { FlatList, View, StyleSheet } from 'react-native';
+import { useSelector } from 'react-redux';
+import { Navigation } from 'react-native-navigation';
+
+import { colors } from '../../../constants/colors';
+import { symbols } from '../../../constants/symbols';
+import { selectFavorites } from '../selectors';
+import { routes } from '../../../constants/routes';
+import { Header } from '../../../components/Header';
+import { Album } from '../../Albums/components/Album';
+import { GeneralNotification } from '../../../components/GeneralNotification';
+
+export const FavoriteAlbums = ({ componentId }) => {
+  const favorites = useSelector(selectFavorites);
+
+  const onOpenTracks = (albumName, id, collectionPrice, artworkUrl60) => {
+    Navigation.push(componentId, {
+      component: {
+        name: routes.FavoriteTracks,
+        passProps: {
+          albumId: id,
+          collectionName: albumName,
+          collectionPrice,
+          artworkUrl60,
+        },
+      },
+    });
+  };
+
+  const combainedFavoriteAlbums = () => {
```

Comment:
This logic should be organized in `selectFavoritesAlbums` function(selectors folder). 

```
const selectFavouriteAlbums = state => { 
    const favorites = state.favoriteAlbumsReducer.favorites;

    return Object.values(favorites).map((favorite) => favorite.album);
 };
```